src/SpotifyApi.NetCore/LibraryApi.cs

```diff
@@ -0,0 +1,410 @@
+
+            var builder = new UriBuilder($"{BaseUrl}/me/albums/contains");
+            builder.AppendToQueryAsCsv("ids", albumIds);
+            return await GetModel<T>(builder.Uri, accessToken);
+        }
+        #endregion
+
+        #region CheckUserSavedShows
+        /// <summary>
+        /// Check if one or more shows is already saved in the current Spotify user’s library.
+        /// </summary>
+        /// <param name="showIds">Required. A comma-separated list of the Spotify IDs for the shows. Minimum: 1 ID. Maximum: 50 IDs.</param>
+        /// <returns>bool[] an array of true or false values, in the same order in which the ids were specified.</returns>
+        /// <remarks>
+        /// https://developer.spotify.com/documentation/web-api/reference/library/check-users-saved-shows/
+        /// </remarks>
+        public async Task<bool[]> CheckUserSavedShows(
+            string[] showIds,
+            string accessToken = null
+            ) => await CheckUserSavedShows<bool[]>(showIds, accessToken);
+
+        /// <summary>
+        /// Check if one or more shows is already saved in the current Spotify user’s library.
+        /// </summary>
+        /// <param name="showIds">Required. A comma-separated list of the Spotify IDs for the shows. Minimum: 1 ID. Maximum: 50 IDs.</param>
+        /// <returns>bool[] an array of true or false values, in the same order in which the ids were specified.</returns>
+        /// <remarks>
+        /// https://developer.spotify.com/documentation/web-api/reference/library/check-users-saved-shows/
+        /// </remarks>
+        public async Task<T> CheckUserSavedShows<T>(
+            string[] showIds,
+            string accessToken = null
+            )
+        {
+            if (showIds?.Length < 1 || showIds?.Length > 50) throw new
+                    ArgumentException("A minimum of 1 and a maximum of 50 show ids can be sent.");
+
+            var builder = new UriBuilder($"{BaseUrl}/me/shows/contains");
+            builder.AppendToQueryAsCsv("ids", showIds);
+            return await GetModel<T>(builder.Uri, accessToken);
+        }
+        #endregion
+
+        #region CheckUserSavedTracks
+        /// <summary>
+        /// Check if one or more tracks is already saved in the current Spotify user’s ‘Your Music’ library.
+        /// </summary>
+        /// <param name="trackIds">Required. A comma-separated list of the Spotify IDs for the tracks. Minimum: 1 ID. Maximum: 50 IDs.</param>
+        /// <returns>bool[] an array of true or false values, in the same order in which the ids were specified.</returns>
+        /// <remarks>
+        /// https://developer.spotify.com/documentation/web-api/reference/library/check-users-saved-tracks/
+        /// </remarks>
+        public async Task<bool[]> CheckUserSavedTracks(
+            string[] trackIds,
+            string accessToken = null
+            ) => await CheckUserSavedTracks<bool[]>(trackIds, accessToken);
```

Comment:
* Method name = `CheckUsersSavedTracks`
* Remove `async` and `await` keywords here.